usage_examples/rel_openai/fewshot.cfg

```diff
@@ -0,0 +1,29 @@
+[paths]
+examples = null
+
+[initialize]
+vectors = "en_core_web_md"
```

Comment:
Can you move all the `[initialize]` blocks to the ends of the configs? (This is the standard config block order and I'm worried we might get confused when maintaining this version in the future.)